discovery-provider/src/solana/solana_client_manager.py

```diff
@@ -48,7 +50,7 @@ def handle_get_sol_tx_info(client, index):
                         f"solana_client_manager.py | get_sol_tx_info | Fetching tx {tx_sig} {endpoint}"
                     )
                     tx_info: ConfirmedTransaction = client.get_confirmed_transaction(
```

Comment:
You should also be able to update this to `get_transaction`

Reply:
nice, done